Index: thunar-shares-plugin-0.2.0/configure.in
===================================================================
--- thunar-shares-plugin-0.2.0.orig/configure.in
+++ thunar-shares-plugin-0.2.0/configure.in
@@ -56,7 +56,7 @@ dnl Packages checks
 dnl -------------------------------------------------------
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.12.0])
 XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.10.0])
-XDT_CHECK_PACKAGE([THUNARX], [thunarx-1], [0.8.0])
+XDT_CHECK_PACKAGE([THUNARX], [thunarx-2], [1.0.0])
 XDT_CHECK_PACKAGE([THUNARVFS], [thunar-vfs-1], [0.8.0])
 
 dnl -------------------------------------------------------
Index: thunar-shares-plugin-0.2.0/thunar-plugin/Makefile.am
===================================================================
--- thunar-shares-plugin-0.2.0.orig/thunar-plugin/Makefile.am
+++ thunar-shares-plugin-0.2.0/thunar-plugin/Makefile.am
@@ -7,7 +7,7 @@ INCLUDES =							\
 	-DDATADIR=\"$(datadir)\"				\
 	$(PLATFORM_CPPFLAGS)
 
-extensionsdir = $(libdir)/thunarx-1
+extensionsdir = $(libdir)/thunarx-2
 extensions_LTLIBRARIES = thunar-shares-plugin.la
 
 thunar_shares_plugin_la_SOURCES =				\
